dotnetv3/S3/TransferAccelerationExample/TransferAcceleration.cs

```diff
@@ -1,28 +1,28 @@
 ﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX - License - Identifier: Apache - 2.0
 
+/// <summary>
+/// Amazon Simple Storage Service (Amazon S3) Transfer Acceleration is a
+/// bucket-level feature that enables you to perform faster data transfers
+/// to Amazon S3. This example shows how to configure  transfer
```

Comment:
2 spaces between 'configure' and 'transfer'

Reply:
fixed